core/src/main/resources/error/README.md

```diff
@@ -1,27 +1,24 @@
 # Guidelines
 
-To throw a standardized user-facing exception, developers should specify the error class and
-message parameters rather than an arbitrary error message.
+To throw a standardized user-facing error of exception, developers should specify the error class
```

Comment:
`error of exception` -> `error or exception`?

Reply:
Whoops, fixed!